settings.py

```diff
@@ -747,21 +747,10 @@
 AVAILABLE_CUSTOM_RULE_ACTIONS = {}
 
 ####### auditcare parameters #######
-AUDIT_VIEWS = [
-    'corehq.apps.settings.views.ChangeMyPasswordView',
-    'corehq.apps.hqadmin.views.users.AuthenticateAs',
-]
-
-AUDIT_MODULES = [
-    'corehq.apps.reports',
-    'corehq.apps.userreports',
-    'corehq.apps.data',
-    'corehq.apps.registration',
-    'corehq.apps.hqadmin',
-    'corehq.apps.accounting',
-    'corehq.apps.cloudcare',
-    'tastypie',
-]
+AUDIT_ALL_VIEWS = False
+AUDIT_VIEWS = []
+AUDIT_MODULES = []
+AUDIT_ADMIN_VIEWS = False
```

Comment:
Just eyeballing it, it seems like this won't in any way change the effective configuration of production or india. Can you/did you sanity check that (maybe just locally) that `AUDIT_ALL_VIEWS = True` in `localsettings.py` overrides these settings in practice? That's the only thing I could imagine going wrong in some unintuitive way

Reply:
There are tests for that https://github.com/dimagi/commcare-hq/blob/699d0f1ef5bcb7a8333e4d6e202bdd83c31cf902/corehq/apps/auditcare/tests/test_middleware.py#L29-L41

I also ran the test you asked for locally and observed new records written to auditcare tables with `AUDIT_ALL_VIEWS = True` in localsettings.py.